aeneas/src/ir/SsaNormalizer.v3

```diff
@@ -1209,15 +1218,13 @@ class SsaRaNormalizer extends SsaRebuilder {
 			if (!isVariant) addNullCheck(oldApp, ninputs[0]);
 			return map0(oldApp);
 		}
-		if (raField.raFacts.RC_UNBOXED) {
-			// field of flattened data type
-			normType(raField.receiver); // XXX: normType() side-effect of flattening
-			var rc = norm.ra.getClass(raField.receiver);
+		normType(raField.receiver); // XXX: normType() side-effect of flattening
```

Comment:
Can you make use of the return value from `normType`?

Reply:
I don't think using the return value here will make much of a difference - and I'll have to cast it to a `VariantNorm` to use it in the if body below anyway (as opposed to just using `rc.variantNorm`)